www/js/lib/uiUtil.js

```diff
@@ -84,12 +84,45 @@ define([], function() {
         return url.replace(regexpRemoveUrlParameters, "$1");
     }
 
+    /**
+     * Displays a Bootstrap warning alert with information about how to access content in a ZIM with unsupported active UI
+     */
+    function displayActiveContentWarning() {
+        // We have to add the alert box in code, because Bootstrap removes it completely from the DOM when the user dismisses it
+        var alertHTML =
+            '<div id="activeContent" class="alert alert-warning alert-dismissible fade in">' +
+                '<a href="#" class="close" data-dismiss="alert" aria-label="close">&times;</a>' +
+                '<strong>Unable to display active content:</strong> This ZIM is not fully supported in jQuery mode.<br />' +
+                'Content may be available by searching above (type a space or a letter of the alphabet), or else ' +
+                '<a id="swModeLink" href="#contentInjectionModeDiv" class="alert-link">switch to Service Worker mode</a> ' +
+                'if your platform supports it. &nbsp;[<a id="stop" href="#displaySettingsDiv" class="alert-link">Permanently hide</a>]' +
+            '</div>';
+        document.getElementById('alertBoxHeader').innerHTML = alertHTML;
+        ['swModeLink', 'stop'].forEach(function(id) {
```

Comment:
It would be good to add a comment here, explaining what the following code does (moving to the configuration section, and highlighting the relevant option)

Reply:
Will do.

Reply:
Commit https://github.com/kiwix/kiwix-js/pull/467/commits/e179c98d3f58553b26b80085fce0f020493decea adds the requested comments.